### PR TITLE
Paper: RQ2

### PR DESCRIPTION
**Pull request by beyond2void**
small update with research question 2

### CODE REVIEW

**Comment by Logende** [comment]
@beyond2void can you refer to the tables within the text? And not just say "yes", but phrase it according to the actual data?

**Comment by beyond2void** [comment]
> @beyond2void can you refer to the tables within the text? And not just say "yes", but phrase it according to the actual data?

Ok, I will do.

**Comment by beyond2void** [comment]
> image 
I connected the research question with our user study table and used some contents as references.

**Comment by PaulBredl** [comment]
@beyond2void I would do some adjustments to the working, ok?

**Comment by beyond2void** [comment]
> @beyond2void I would do some adjustments to the working, ok?

Yes, no problem.

**Comment by beyond2void** [comment]
maybe you can approve it once you finished improvements. @PaulBredl

File: paper/user_study.tex [hunk 1]
```diff
@@ -81,16 +81,22 @@ \subsubsection{\textbf{RQ1:} Which aspects of the tool can be improved?} %Keyuri
...
 With the help of hints, all tasks could be completed by every participant.
+\begin{itemize}
+    \item \textbf{RQ2.1}: Table~\ref{table:results} shows that all
+    participants were able to retrieve information related to the schema and data.
+    All participants were using mostly the GUI panel to retrieve information.
+    Difficulties occurred when participants did not know that they could use the tooltips to get more information about the properties.
+    \item \textbf{RQ2.2}: All participants were able to modify the given test file according to the given tasks.
+    For some (User study 1 and 4) it was challenging to find a specific property, and they were not aware of the search functionality.
+    \item \textbf{RQ2.3}: Most of the users are able to modify the schema file according to the tasks.
+    The biggest difficulty was that participants did not know how to add new property in schema editor using the GUI panel.
```
Comment (PaulBredl) [inline paper/user_study.tex:96]:
```suggestion
    The biggest difficulty was that participants did not know how to add a new property in the schema editor using the GUI panel.
```

**Review by PaulBredl — APPROVED** [review]

**Pull request merged**